Sources/SUIComponents/Organisms/InformationMessageCard.swift

```diff
@@ -90,19 +93,15 @@ extension Components.Organisms {
 
         @ViewBuilder func primary(cta: CTA) -> AnyView {
             AnyView(
-                Button(cta.message) {
-                    print("primary button tapped")
-                }
+                Button(cta.message, action: handler)
```

Comment:
I see both primary and secondary CTA call the same handler. Is it always the case that an information card will only have one CTA (which may be primary or secondary)?

Reply:
I believe so based on the existing covertLinks in `InformationMessageViewModelTransformer` which only converts to one displayType which figures out primary/secondary.